src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -172,158 +170,6 @@ public String getIdentifierQuoteString() throws SQLException {
-                + "JSON,"
-                + "KEYS,"
-                + "KILL,"
-                + "LIMIT,"
-                + "LOCK,"
-                + "LOGS,"
-                + "LONGTEXT,"
-                + "LOW_PRIORITY,"
-                + "MASTER,"
-                + "MEDIUMBLOB,"
-                + "MEDIUMTEXT,"
-                + "MICROSECOND,"
-                + "MINUS,"
-                + "MINUTE_MICROSECOND,"
-                + "MINUTE_SECOND,"
-                + "MODIFY,"
-                + "MUTEX,"
-                + "OBJECTID,"
-                + "OFF,"
-                + "OFFSET,"
-                + "OJ,"
-                + "PARTITIONS,"
-                + "PLUGINS,"
-                + "PROCESSLIST,"
-                + "PROFILE,"
-                + "PROFILES,"
-                + "PROXY,"
-                + "QUARTER,"
-                + "QUERY,"
-                + "REGEXP,"
-                + "RELAYLOG,"
-                + "RENAME,"
-                + "RLIKE,"
-                + "SAMPLE,"
-                + "SCHEMAS,"
-                + "SECOND_MICROSECOND,"
-                + "SEPARATOR,"
-                + "SERIAL,"
-                + "SHOW,"
-                + "SIGNED,"
-                + "SLAVE,"
-                + "SQL_BIGINT,"
-                + "SQL_DATE,"
-                + "SQL_DOUBLE,"
-                + "SQL_TIMESTAMP,"
-                + "SQL_TSI_DAY,"
-                + "SQL_TSI_HOUR,"
-                + "SQL_TSI_MINUTE,"
-                + "SQL_TSI_MONTH,"
-                + "SQL_TSI_QUARTER,"
-                + "SQL_TSI_SECOND,"
-                + "SQL_TSI_WEEK,"
-                + "SQL_TSI_YEAR,"
-                + "SQL_VARCHAR,"
-                + "STATUS,"
-                + "STORAGE,"
-                + "STRAIGHT_JOIN,"
-                + "SUBDATE,"
-                + "SUBSTR,"
-                + "TABLES,"
-                + "TEXT,"
-                + "TIMESTAMPADD,"
-                + "TIMESTAMPDIFF,"
-                + "TINYINT,"
-                + "TINYTEXT,"
-                + "TRADITIONAL,"
-                + "TRIGGERS,"
-                + "UNLOCK,"
-                + "UNSIGNED,"
-                + "USE,"
-                + "UTC_DATE,"
-                + "UTC_TIMESTAMP,"
-                + "VARIABLES,"
-                + "WARNINGS,"
-                + "WEEK,"
-                + "XOR,"
-                + "YEAR_MONTH";
-    }
-
-    @Override
-    public String getNumericFunctions() throws SQLException {
-        return MongoFunction.numericFunctionsString;
-    }
-
-    @Override
-    public String getStringFunctions() throws SQLException {
-        return MongoFunction.stringFunctionsString;
-    }
-
-    @Override
-    public String getSystemFunctions() throws SQLException {
-        return "DATABASE,USER,SYSTEM_USER,SESSION_USER,VERSION";
-    }
-
-    @Override
-    public String getTimeDateFunctions() throws SQLException {
-        return MongoFunction.dateFunctionsString;
-    }
-
     @Override
```

Comment:
Anything returning a `MySQLResultSet` needs to be made abstract and overridden in the subclass. I guess I missed a few in the design doc!

Reply:
Ok I'll look through and make sure I catch all of these! I can also add them to the design doc.

Reply:
done